### DISCUSSION TITLE
New KadeEngine's source code for Release 1.4.1 still uses the Nightly2 version. Any ideas why?

### DISCUSSION THREAD

**Discussion by ProjectAnzo** [KadeArchive/Kade-Engine#172] [Q&A]
I don't know if it's just me, but when I compiled the code from the Release 1.4.1 source code, it uses the Nightly2 version of the source code. And mostly everything that was added from Nightly2 and not the actual release. Hopefully, this is a minor mistake, so the 1.4.1 release source code would feature the simple/complex accuracy system.
![kadeengine folders](https://user-images.githubusercontent.com/81794723/114267806-bd0e6e00-99b2-11eb-8eba-dec2b3b845c0.PNG)
![nightly code](https://user-images.githubusercontent.com/81794723/114267807-bda70480-99b2-11eb-8e8c-0596bd6b8e83.PNG)

**Comment by puyoxyz** [comment]
hehe looks like @kadedev forgot to change the version number

**Reply by ProjectAnzo** [reply]
but like it actually was the nightly2 version lol

**Reply by puyoxyz** [reply]
oh then maybe u downloaded the wrong thing or kade fricked up the release
